contracts/stellar-axelar-gateway/src/storage.rs

```diff
@@ -2,22 +2,21 @@ use soroban_sdk::{contracttype, BytesN, String};
 use stellar_axelar_std::contractstorage;
 
 #[contracttype]
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct MessageApprovalKey {
     pub source_chain: String,
     pub message_id: String,
 }
 
 #[contracttype]
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(PartialEq)]
 pub enum MessageApprovalValue {
     NotApproved,
     Approved(BytesN<32>),
     Executed,
 }
 
 #[contractstorage]
-#[derive(Clone, Debug)]
 pub enum DataKey {
```

Comment:
```suggestion
enum DataKey {
```
DataKey should be private now